components/AppName/index.js

```diff
@@ -0,0 +1,15 @@
+import { styled } from "styled-components";
+
+export default function AppName() {
+  return <StyledName>TripPic</StyledName>;
+}
+
+const StyledName = styled.h1`
```

Comment:
The naming could be a bit confusing in the future, maybe Name the h1 headline like this: StyledAppName.